debian/copyright

```diff
@@ -0,0 +1,34 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ndn-cxx
+Source: http://named-data.net
+
+Files: *
+Copyright: 2010-2016 Alexander Afanasyev   <http://lasr.cs.ucla.edu/afanasyev/index.html>
+           2010-2016 Yingdi Yu             <http://irl.cs.ucla.edu/~yingdi/>
+           2010-2016 Junxiao Shi           <http://www.cs.arizona.edu/people/shijunxiao/>
+           2010-2016 Jeff Thompson         <jefft0@remap.ucla.edu>
+           2010-2016 Zhenkai Zhu           <http://irl.cs.ucla.edu/~zhenkai/>
+License: LGPL-3.0+
+
+Files: debian/*
+Copyright: 2016 Endless Computers INC
+Copyright: 2016 Niv Sardi <xaiki@endlessm.com>
```

Comment:
`Endless Mobile Inc` only. While I understand the idea of adding all people that wrote the code here (and it's a valid one), at Endless the copyright itself is owned by the company and not the individual afaik.